src/js/state/View/reducer.js

```diff
@@ -8,20 +8,14 @@ const init: ViewState = {
   leftSidebarWidth: 350,
   resultsTab: null,
   timeZone: "UTC",
-  investigationView: "linear",
-  isIngesting: false
```

Comment:
I have made changes to the state format here. I removed this `isIngesting` boolean that was not being read from anywhere in the application. I think we are safe here and do not need to migrate or increment the version number on the state. But I want reviewers to double check me here.

Reply:
I did a check w/ master vs. this branch and didn't run into any problems.